apps/schematic/api/schematic_api/controllers/storage_controller_impl.py

```diff
@@ -21,10 +56,59 @@ def list_storage_project_datasets(project_id):  # noqa: E501
         )
         res = page
         status = 200
+    except SynapseAuthenticationError as error:
+        status = 401
+        res = ("Internal error", status, str(error))
+
     # except DoesNotExist:
     #     status = 404
     #     res = BasicError("The specified resource was not found", status)
     except Exception as error:
         status = 500
         res = BasicError("Internal error", status, str(error))
     return res, status
+
+
+def list_storage_project_manifests(project_id, asset_view):
+    """List manifests in a given storage project"""
+    # load config
+    config_handler(asset_view=asset_view)
+
+    # get access token
+    bearer_token = get_access_token()
+
+    # load token to synapse storage
+    try:
+        store = SynapseStorage(access_token=bearer_token)
+        lst_storage_projects = store.getProjectManifests(projectId=project_id)
+
+        page = ManifestsPage(
+            number=0,
+            size=100,
+            total_elements=len(lst_storage_projects),
+            total_pages=1,
+            has_next=False,
+            has_previous=False,
+            manifests=lst_storage_projects,
+        )
+        res = page
+        status = 200
+
+    except Exception as error:
+        status = 500
+        res = BasicError("Internal error", status, str(error))
+
+    except SynapseAuthenticationError as error:
+        status = 403
+        res = BasicError(
+            "Synapse authentication error. Unauthorized access.", status, str(error)
```

Comment:
We could use the keyword "forbidden" to avoid confusion with the 401 error (Unauthorized).

```suggestion
            "Forbidden Synapse access error", status, str(error)
```